helm/dagster/values.yaml

```diff
@@ -1142,3 +1142,13 @@ serviceAccount:
   name: ""
 
   annotations: {}
+
+retention:
+  schedule:
+    purgeAfterDays: -1
+  sensor:
+    purgeAfterDays:
+      failure: -1
+      skipped: 7
+      started: -1
+      success: -1
```

Comment:
should probably be nested under `dagsterDaemon`. documentation for each field would be nice too. Adding a link to https://docs.dagster.io/deployment/dagster-instance#data-retention would be great as well

Reply:
Added some documentation, but resisting nesting under dagsterDaemon, which feels more like an implementation detail.  Also, we might introduce new fields with data retention policies that don't have to do with schedule/sensor ticks.